package.json

```diff
@@ -8,5 +8,8 @@
   "license": "MIT",
   "devDependencies": {
     "mocha": "^5.2.0"
-  }
+  },
+	"scripts": {
```

Comment:
this file is using two space indents.

please do the same.

Reply:
Sorry about that. I was sure my current vim install was using spaces instead of tabs. Fixing it just now.